docs/runtimes/configuring-job-resources.md

```diff
@@ -237,22 +231,21 @@ And another function that can only be scheduled on preemptible nodes:
 import mlrun
 import os
 
-train_fn = mlrun.code_to_function('training', 
+train_fn = mlrun.set_function('training', 
                             kind='job', 
                             handler='my_training_function') 
 train_fn.with_preemption_mode(mode="prevent") 
```

Comment:
```suggestion
train_fn.with_preemption_mode(mode="constrain") 
```

To have a function that can only be scheduled on preemptible nodes, it needs to be `constrain`.